test/pseudo-tty/stdin-setrawmode.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+
+process.stdin.setRawMode(true);
+assert(process.stdin.isRaw);
+
+process.stdin.setRawMode(false);
+assert(!process.stdin.isRaw);
```

Comment:
Perhaps you can use assert.strictEqual instead of plain assert, that would catch truthy-but-not-boolean values.